developer-tools.md

```diff
@@ -397,6 +397,18 @@ Other tips:
 - "Rebuild Project" can fail the first time the project is compiled, because generate source files 
 are not automatically generated. Try clicking the "Generate Sources and Update Folders For All 
 Projects" button in the "Maven Projects" tool window to manually generate these sources.
+- Maven bundled in IntelliJ may not meet the minimum version requirement of the Spark. If that happens,
+the action "Generate Sources and Update Folders For All Projects" could fail silently. If you saw error like
+``` 
+2019-04-14 16:05:24,796 [ 314609]   INFO -      #org.jetbrains.idea.maven - [WARNING] Rule 0: org.apache.maven.plugins.enforcer.RequireMavenVersion failed with message:
+Detected Maven Version: 3.3.9 is not in the allowed range 3.6.0.
```

Comment:
Hi, @William1104 . Could you remove this verbose error message dump? 
In general, it's not a good idea to include the whole error message. This message is already outdated because Apache Spark already moves to Maven 3.6.1 today.

Reply:
Opps. It's already updated. Thanks.

Reply:
Yes. Agree and i am going to remove it.